protocols/LifeGeneTEST/plate_filling.ot2.py

```diff
@@ -0,0 +1,66 @@
+from opentrons import labware, instruments
```

Comment:
We don't really need this protocol, it was a test for staging-a

Reply:
will remove